src/screens/EditPlaylistScreen.tsx

```diff
@@ -159,16 +207,37 @@ export class EditPlaylistScreen extends React.Component<Props, State> {
     return <SortableListRow active={active} cell={cell} />
   }
 
+  _handleRemovePlaylistItemPress = async (item: any) => {
+    const { playlist, sortableListData } = this.state
+
+    this.setState({ isRemoving: true }, async () => {
+      try {
+        const episodeId = !item.startTime && item.id
+        const mediaRefId = (item.startTime || item.startTime === 0) && item.id
```

Comment:
if one of the first conditions resolves to false, isn't mediaRefId going to be just "false"? Is that ok with the addOrRemovePlaylistItem function?

Reply:
It's ok because the addOrRemovePlaylistItem service won't pass mediaRefId if it is falsey. In any case I replaced this logic with a ternary so it's clearer.